shell/plugins/dashboard-store/actions.js

```diff
@@ -184,6 +186,13 @@ export default {
 
     console.log(`Find All: [${ ctx.state.config.namespace }] ${ type }`); // eslint-disable-line no-console
     opt = opt || {};
+    // we should always exclude managed fields when retreiving resources to cut down on payload size and store memory
+    if (isArray(opt?.excludeFields)) {
+      partial = true;
+      opt.excludeFields = [...opt.excludeFields, 'metadata.managedFields'];
```

Comment:
I went back and forth on this actually, but the spec on this is that we should always exclude metadata.managedFields. Moreover, since we don't use managedFields anywhere, there really isn't a case where we wouldn't want to exclude it and making the excludeFields option override it would simply result in more boilerplate as every developer providing an "excludeFields" array would need to remember to include it.

Reply:
The spec is we set it by default, not that it's always there. It seems odd to let the user specify their own but not use what they've request, which could be specifically to include managedFields. 

You're right though, we don't have a use case for setting excludes fields. whatever we hit first should be used to drive a change here